packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
These are Visual studio files which got added , have removed them in the next commit.